Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -2564,22 +2600,6 @@ def main():
             return_results(results=get_console_version(client=client))
         elif requested_command == 'prisma-cloud-compute-custom-feeds-ip-list':
             return_results(results=get_custom_feeds_ip_list(client=client))
-        elif requested_command == 'prisma-cloud-compute-profile-host-list':
-            return_results(results=get_profile_host_list(client=client, args=demisto.args()))
-        elif requested_command == 'prisma-cloud-compute-profile-container-list':
-            return_results(results=get_container_profile_list(client=client, args=demisto.args()))
-        elif requested_command == 'prisma-cloud-compute-profile-container-hosts-list':
-            return_results(results=get_container_hosts_list(client=client, args=demisto.args()))
-        elif requested_command == 'prisma-cloud-compute-profile-container-forensic-list':
-            return_results(results=get_profile_container_forensic_list(client=client, args=demisto.args()))
-        elif requested_command == 'prisma-cloud-compute-host-forensic-list':
-            return_results(results=get_profile_host_forensic_list(client=client, args=demisto.args()))
-        elif requested_command == 'prisma-cloud-compute-custom-feeds-ip-add':
-            return_results(results=add_custom_ip_feeds(client=client, args=demisto.args()))
-        elif requested_command == 'prisma-cloud-compute-console-version-info':
-            return_results(results=get_console_version(client=client))
-        elif requested_command == 'prisma-cloud-compute-custom-feeds-ip-list':
```

Comment:
how is this deletion related to the PR?

Reply:
These lines are duplicate with the lines above it, I assume it happened by mistake so I fixed it here. As I've worked on those commands I noticed it and removed these lines.